src/actions/guides/testing/testing_actions.cr

```diff
@@ -0,0 +1,16 @@
+class Guides::Testing::TestingActions < GuideAction
+  guide_route "/testing/testing-actions"
+
+  def self.title
+    "Testing Actions"
+  end
+
+  def markdown : String
+    <<-MD
+    ## HTML Actions
+
+    ## JSON Actions
```

Comment:
I think JSON should come first sine it is more likely you'll test html actions with luckyflow and json would be this way. Thoughts?

Reply:
Also thoughts on changing this to `Testing JSON responses`?